stripe/src/main/java/com/stripe/android/view/AddSourceActivity.java

```diff
@@ -0,0 +1,159 @@
+package com.stripe.android.view;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.annotation.NonNull;
+import android.support.annotation.VisibleForTesting;
+import android.support.transition.Fade;
+import android.support.transition.TransitionManager;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.FrameLayout;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import com.stripe.android.PaymentConfiguration;
+import com.stripe.android.R;
+import com.stripe.android.SourceCallback;
+import com.stripe.android.Stripe;
+import com.stripe.android.model.Card;
+import com.stripe.android.model.Source;
+import com.stripe.android.model.SourceParams;
+
+public class AddSourceActivity extends AppCompatActivity {
+
+    public static final String EXTRA_NEW_SOURCE = "new_source";
```

Comment:
why are `EXTRA_NEW_SOURCE` and `EXTRA_SHOW_ZIP` prefixed with `EXTRA_`? just wondering :)